### DISCUSSION TITLE
segwit vs taproot address

### DISCUSSION THREAD

**Discussion by skyrover7** [ordinals/ord#1772] [Q&A]
Seen a few folks in the community and in the ordinal discord asking about receiving inscriptions on `bc1q` addresses. While this seems possible to do, are inscriptions sent to those addresses stuck? lost? If so, any recourse?

example transaction here: https://ordinals.com/tx/ce16281687f8a534900fa0e34edec68907520e8d99cc11990303a4a6dbdc2924

**Comment by lilsats** [comment]
https://github.com/casey/ord/discussions/1753#discussioncomment-5000704

According to this it should be fine as long as you are mindful not to lose the inscribed sat to fees
